h2o-core/src/test/java/water/fvec/NewChunkSpeedTest.java

```diff
@@ -0,0 +1,266 @@
+package water.fvec;
+
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+import water.MRTask;
+import water.Scope;
+import water.TestUtil;
+import water.util.Log;
+
+import static org.junit.Assert.assertTrue;
+
+/***
+ * This test is written to measure the speed with which NewChunks are written for various data types:
+ * integer, long and double
+ */
+
+public class NewChunkSpeedTest extends TestUtil {
+  @BeforeClass() public static void setup() { stall_till_cloudsize(1); }
+  int rowNumber = 1000000;
+  int rowInterval = 1000;
+  double tolerance = 1e-10;
+  int numberLoops=2;
+
+  @Test public void testParseDoublesConst(){
+    double startTime = System.currentTimeMillis();
+    for (int index=0; index<numberLoops; index++)
+      testsForDoubles(true, false, false);
+    double endTime = (System.currentTimeMillis()-startTime)*0.001;  // change time to seconds
+    Log.info("New Chunk test for constant doubles:", " time(s) taken for "+numberLoops+" loops is "+endTime);
+  }
+
+  @Test public void testParseBigDoublesConst(){
+    double startTime = System.currentTimeMillis();
+    for (int index=0; index<numberLoops; index++)
+      testsForDoubles(true, true, false);
+    double endTime = (System.currentTimeMillis()-startTime)*0.001;  // change time to seconds
+    Log.info("New Chunk test for big constant doubles:", " time(s) taken for "+numberLoops+" loops is "+endTime);
```

Comment:
you can use https://github.com/h2oai/h2o-3/blob/master/h2o-core/src/main/java/water/util/PrettyPrint.java to print time.

Reply:
Thanks, I have incorporated it.